cmd/libsnap-confine-private/utils.h

```diff
@@ -39,6 +39,48 @@ bool sc_is_debug_enabled(void);
  **/
 bool sc_is_reexec_enabled(void);
 
+/**
+ * sc_identity describes the user performing certain operation.
+ *
+ * UID and GID may represent actual user and group accounts. Either value may
+ * also be set to -1 to indicate that a specific identity change should not
+ * be performed.
+**/
+typedef struct sc_identity {
+	uid_t uid;
+	gid_t gid;
+	unsigned change_uid:1;
```

Comment:
Why not bool? Not sure we're saving much here.

Reply:
This is my personal preference. Note that we are not saving anything anyway due to alignment. This way alignment no longer adds padding.